src/components/ProjectCard.tsx

```diff
@@ -12,17 +14,14 @@ interface ProjectCardProps {
 export default function ProjectCard({ project }: ProjectCardProps) {
   const navigate = useNavigate();
   const [editOpen, setEditOpen] = useState(false);
+  const [deleteOpen, setDeleteOpen] = useState(false);
+  const [deleting, setDeleting] = useState(false);
```

Comment:
niepotrzebne. jak zrobisz na to hook to `useMutation` bedzie mialo w sobie `isLoading`